app/bundles/EmailBundle/Views/Email/list.html.twig

```diff
@@ -0,0 +1,226 @@
+{% endif %}
+
+{% block content %}
+
+{% if items|length > 0 %}
+
+<div class="table-responsive">
+    <table class="table table-hover table-striped table-bordered email-list">
+        <thead>
+        <tr>
+            {{- include('MauticCoreBundle:Helper:tableheader.html.twig', {
+                'checkall'        : 'true',
+                'routeBase'       : 'email',
+                'templateButtons' : {
+                    'delete' : permissions['email:emails:deleteown'] or permissions['email:emails:deleteother']
+                },
+            }) -}}
+
+            {{- include('MauticCoreBundle:Helper:tableheader.html.twig', {
+                'sessionVar' : 'email',
+                'orderBy'    : 'e.name',
+                'text'       : 'mautic.core.name',
+                'class'      : 'col-email-name',
+            }) -}}
+
+            {{- include('MauticCoreBundle:Helper:tableheader.html.twig', {
+                'sessionVar' : 'email',
+                'orderBy'    : 'c.title',
+                'text'       : 'mautic.core.category',
+                'class'      : 'visible-md visible-lg col-email-category',
+            }) -}}
+
+            <th class="visible-sm visible-md visible-lg col-email-stats">{{ 'mautic.core.stats'|trans }}</th>
+
+            {{- include('MauticCoreBundle:Helper:tableheader.html.twig', {
+                'sessionVar' : 'email',
+                'orderBy'    : 'e.dateAdded',
+                'text'       : 'mautic.lead.import.label.dateAdded',
+                'class'      : 'visible-lg col-email-dateAdded',
+            }) -}}
+
+            {{- include('MauticCoreBundle:Helper:tableheader.html.twig', {
+                'sessionVar' : 'email',
+                'orderBy'    : 'e.dateModified',
+                'defaultDir' : 'DESC',
+                'text'       : 'mautic.lead.import.label.dateModified',
+                'class'      : 'visible-lg col-email-dateModified',
+                'default'    : true,
+            }) -}}
+
+            {{- include('MauticCoreBundle:Helper:tableheader.html.twig', {
+                'sessionVar' : 'email',
+                'orderBy'    : 'e.createdByUser',
+                'text'       : 'mautic.core.createdby',
+                'class'      : 'visible-lg col-email-createdByUser',
+            }) -}}
+
+            {{- include('MauticCoreBundle:Helper:tableheader.html.twig', {
+                'sessionVar' : 'email',
+                'orderBy'    : 'e.id',
+                'text'       : 'mautic.core.id',
+                'class'      : 'visible-md visible-lg col-email-id',
+            }) -}}
+        </tr>
+        </thead>
+        <tbody>
+        {% for item in items %}
+        {% set hasVariants  = item.isVariant() %}
+        {% set hasTranslations  = item.isTranslation() %}
+        {% set type = item.getEmailType() %}
+        {% set mauticTemplateVars = mauticTemplateVars|merge({'item' : item}) %}
```

Comment:
I'm getting the error `500 Internal Server Error - Variable "mauticTemplateVars" does not exist.`

Update to: `{% set mauticTemplateVars = mauticTemplateVars|default([])|merge({'item' : item}) %}`